.github/workflows/bump-version.sh

```diff
@@ -9,5 +9,5 @@ fi
 FROM_VERSION=$1
 TO_VERSION=$2
 
-sed --in-place "s/version = '${FROM_VERSION}'/version = '${TO_VERSION}'/g" build.gradle
-git commit -m "Version: bump ${TO_VERSION}" build.gradle
+sed --in-place "s/version=${FROM_VERSION}/version=${TO_VERSION}/g" gradle.properties
```

Comment:
I'm not able to run this script locally for some reason (sed error), but curious why you removed the spaces and single quotes.  Are you able to test it?

Reply:
The string in `build.gradle` that was replaced looks like:

```
version = '5.4.0-SNAPSHOT'
```

Where as in `gradle.properties` it looks like:

```
version=5.4.0-SNAPSHOT
```

I tested without the `--in-place` flag thats not supported on mac osx sed eg:

```sed  "s/version=5.4.0-SNAPSHOT/version=5.4.0/g" gradle.properties```

Reply:
Ah, I see.  Thanks!

Reply:
but should `-i` make more sense for it is universally supported? `--in-place` is a GNU alternative long argument but `-i` is always working (and MAC user could test it without trouble).

Reply:
nvm. Just found on mac we need special treatment as:
```
sed -i '' 's/STRING_TO_REPLACE/STRING_TO_REPLACE_IT/g' filename
```
then all good. no need to change.

Reply:
Even with `-i` on mac it still needs work.

I believe this will work: 
```
sed -i '' 's/version=5.4.0-SNAPSHOT/version=5.4.0/g' gradle.properties
```

However, I don't have the ability to test on both.